glaciers/glaciers.ipynb

```diff
@@ -317,21 +191,40 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "header = pn.Row(pn.panel(oggm_logo),  pn.layout.Spacer(width=30), \n",
-    "                pn.Column(pn.panel(title, width=400), pn.panel(instruction, width=500)),\n",
-    "                pn.layout.HSpacer(), pn.Column(dynamic_count, pn.layout.Spacer(height=20), clear_button), \n",
-    "                pn.panel(pn_logo, width=140))\n",
+    "instruction = pn.pane.Markdown(\"\"\"\n",
+    "This dashboard visualizes all global glaciers and allows exploring the relationships \n",
+    "between their locations and variables such as their elevation, temperature and annual precipitation.\n",
+    "<br>Box- or lasso-select on each plot to subselect and hit the \"Clear selection\" button to reset. \n",
+    "See the notebook source code for how to build apps like this!\"\"\", width=250)\n",
+    "\n",
+    "panel_logo = pn.pane.PNG(\n",
+    "    'https://panel.holoviz.org/_static/logo_stacked.png',\n",
+    "    link_url='https://panel.holoviz.org', width=250, align='center'\n",
+    ")\n",
     "\n",
-    "content = pn.Column(header, pn.Row(geomap, elevation), pn.Row(temperature, precipitation), width_policy='max', height_policy='max')\n",
+    "oggm_logo = pn.pane.PNG(\n",
+    "    'https://raw.githubusercontent.com/OGGM/oggm/master/docs/_static/logos/oggm_s_alpha.png',\n",
```

Comment:
We should probably not rely on external links like that. How about saving that picture in the examples folder and link it relatively?

Reply:
OK.